content/notifications/send-notifications/get-started/user-guide/_index.en.md

```diff
@@ -13,8 +13,18 @@ In the meantime existing guidelines for sending notifications through Altinn 2 a
 [Please familiarize yourself with existing documentation and guidelines](https://altinn.github.io/docs/utviklingsguider/varsling/)
 {{% /notice %}}
 
-
-
+## Notification
+### SMS
+- The mobile number must contain the country code, preferably with "+", while "00" would also work (e.g., +47900XXXXX, 0047900XXXXX).
+- SMS to 5-digit numbers is not supported.
+- Norwegian mobile numbers must start with "4" or "9" after the country code "+47."
+  - Valid Mobile Number Examples: +47400XXXXX, +47900XXXXX, 0047400XXXXX, 0047900XXXXX.
+  - Invalid Mobile Number Examples: +47500XXXXX, +47600XXXXX, 0047500XXXXX, 0047600XXXXX.
+
+# International Mobile Number
```

Comment:
![image](https://github.com/Altinn/altinn-studio-docs/assets/47737608/9aa3f8ca-56cb-4060-89a0-8ac35d6fbdbe)

heading here seems off. I guess it should be a sub section of SMS ?

Reply:
Making the heading a little smaller than the SMS as it is a subset of SMS.